samples/nrf_desktop/configuration/nrf52840_pca10059/app_ZReleaseMCUBoot.conf

```diff
@@ -13,6 +13,7 @@ CONFIG_DESKTOP_LED_COUNT=2
 CONFIG_DESKTOP_LED_COLOR_COUNT=1
 
 CONFIG_DESKTOP_USB_ENABLE=y
+CONFIG_DESKTOP_ERROR_REBOOT_TIMEOUT=120
```

Comment:
can we set the default value to 120 to not repeat it?

Reply:
ok